ndcube/asdf/resources/schemas/ExtraCoords-0.1.0.yaml

```
@@ -0,0 +1,37 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/ExtraCoords-0.1.0"
+
+title:
+  Represents the ndcube ExtraCoords object
+
+description:
+  Represents the ndcube ExtraCoords object
+
+type: object
+properties:
+  wcs:
+    tag: "tag:stsci.edu:gwcs/wcs-1.*"
+  mapping:
+    type: array
+  lookup_tables:
+    type: array
+    items:
+        type: array
+        items:
+            - oneOf:
+              - type: number
+              - type: array
+            - oneOf:
+              - $ref: "asdf://sunpy.org/ndcube/schemas/QuantityTableCoordinate-0.1.0"
+              - $ref: "asdf://sunpy.org/ndcube/schemas/SkyCoordTableCoordinate-0.1.0"
+              - $ref: "asdf://sunpy.org/ndcube/schemas/TimeTableCoordinate-0.1.0"
+  dropped_tables:
+    type: array
+  ndcube:
+    tag: tag:sunpy.org:ndcube/ndcube/NDCube-0.1.0
```

We'll have to go through and update all of these once the schemas are "stable" at 1.0.0. We might as well start using the wildcard now.
```suggestion
    tag: tag:sunpy.org:ndcube/ndcube/ndcube-0.*
```